tests/core/test_core.py

```diff
@@ -12,105 +12,19 @@
-        assert _OrchestratorFactory._orchestrator == _Orchestrator
-        assert orchestrator._dispatcher is not None
-        assert isinstance(orchestrator._dispatcher, _DevelopmentJobDispatcher)
-        assert isinstance(_OrchestratorFactory._dispatcher, _DevelopmentJobDispatcher)
-        orchestrator.stop()
-
-    def test_run_orchestrator_as_a_service_standalone_mode(self):
-        _OrchestratorFactory._dispatcher = None
-
-        orchestrator = Orchestrator()
-        assert orchestrator._orchestrator is None
-
-        assert orchestrator._dispatcher is None
-        assert _OrchestratorFactory._dispatcher is None
-
-        Config.configure_job_executions(mode=JobConfig._STANDALONE_MODE, max_nb_of_workers=2)
-        orchestrator.run()
-        assert orchestrator._orchestrator is not None
-        assert orchestrator._orchestrator == _Orchestrator
-        assert _OrchestratorFactory._orchestrator is not None
-        assert _OrchestratorFactory._orchestrator == _Orchestrator
-        assert orchestrator._dispatcher is not None
-        assert isinstance(orchestrator._dispatcher, _StandaloneJobDispatcher)
-        assert isinstance(_OrchestratorFactory._dispatcher, _StandaloneJobDispatcher)
-        assert orchestrator._dispatcher.is_running()
-        assert _OrchestratorFactory._dispatcher.is_running()
-        orchestrator.stop()
+class TestCore:
+    def test_run_core_with_depracated_message(self, caplog):
+        with pytest.warns(DeprecationWarning):
+            core = Core()
+        core.run()
 
-    def test_orchestrator_service_can_only_be_run_once(self):
-        orchestrator_instance_1 = Orchestrator()
-        orchestrator_instance_2 = Orchestrator()
-
-        orchestrator_instance_1.run()
-
-        with pytest.raises(OrchestratorServiceIsAlreadyRunning):
-            orchestrator_instance_1.run()
-        with pytest.raises(OrchestratorServiceIsAlreadyRunning):
-            orchestrator_instance_2.run()
-
-        # Stop the Orchestrator service and run it again should work
-        orchestrator_instance_1.stop()
-
-        orchestrator_instance_1.run()
-        orchestrator_instance_1.stop()
-        orchestrator_instance_2.run()
-        orchestrator_instance_2.stop()
-
-    def test_block_config_update_when_orchestrator_service_is_running_development_mode(self):
-        _OrchestratorFactory._dispatcher = None
-
-        orchestrator = Orchestrator()
-        orchestrator.run()
-        with pytest.raises(ConfigurationUpdateBlocked):
-            Config.configure_data_node(id="i1")
-        orchestrator.stop()
-
-    @pytest.mark.standalone
-    def test_block_config_update_when_orchestrator_service_is_running_standalone_mode(self):
-        _OrchestratorFactory._dispatcher = None
+        assert isinstance(core, Orchestrator)
+        expected_message = (
+            "Class `Core` has been deprecated, an object of class `Orchestrator` will be created instead."
```

Comment:
I believe we need to update this test message as well

Reply:
haha you're absolutely spotted on!! I just did and pushed the change also :D